src/Themes/DigitalTheme/Views/OrchardCore.Users/Account/Login.cshtml

```diff
@@ -0,0 +1,99 @@
+@using Microsoft.AspNetCore.Identity
+@using OrchardCore.Entities
+@using OrchardCore.Settings
+@using OrchardCore.Users
+@using OrchardCore.Users.Models
+@using OrchardCore.DisplayManagement.Razor
+@model OrchardCore.Users.ViewModels.LoginViewModel
+@inject SignInManager<IUser> SignInManager
+@inject ISiteService SiteService
+@inject UserManager<IUser> UserManager;
+@{
+    ViewLayout = "Layout__Login";
+
+    var userCanRegister = (await SiteService.GetSiteSettingsAsync()).As<RegistrationSettings>().UsersCanRegister == UserRegistrationType.AllowRegistration;
+    var allowResetPassword = (await SiteService.GetSiteSettingsAsync()).As<ResetPasswordSettings>().AllowResetPassword;
+    var loginProviders = (await SignInManager.GetExternalAuthenticationSchemesAsync()).ToList();
+    var disableLocalLogin = (await SiteService.GetSiteSettingsAsync()).As<LoginSettings>().DisableLocalLogin;
+}
+    <h1 class="login">@T["Log in"]</h1>
+    <p class="login h4"><a href="@Url.Content("~/")"><i class="icon fas fa-cubes mr-2"></i>@(Site.SiteName)</a></p>
+    <div class="login row">
+        @if (loginProviders.Count > 0 || userCanRegister)    
+        {
+        <div class="flex-fill col-lg-4 col-md-6 col-12">
+            @if (loginProviders.Count > 0) {     
+            <section class="section-panel mb-4">
+                <h2 class="text-center mb-3">@T["Log in with"]</h2>
+                <form asp-controller="Account" asp-action="ExternalLogin" asp-route-returnurl="@ViewData["ReturnUrl"]" method="post" class="d-flex justify-content-center">
+                    <div class="d-flex flex-column justify-content-center">
+                        @foreach (var provider in loginProviders)
+                        {
+                            <button type="submit" class="btn btn-lg btn-primary m-2 nav-link is-rounded" name="provider" value="@provider.Name" title="@provider.DisplayName">
+                                @if (provider.DisplayName == "GitHub") { <i class="fab fa-github mr-2"></i> } 
```

Comment:
Think about refactoring this into a tag helper or liquid filter at some point

Reply:
Will do. Created a new issue for it.